wordcloud_test.py

```diff
@@ -51,11 +51,11 @@ def test_word_cloud():
 
     colours = [(255, 0, 0), (255, 127, 0), (255, 255, 0), (0, 255, 0), (0, 0, 255), (75, 0, 130),
                (148, 0, 211)]  # This is a list of RGB codes for the colours of the rainbow
-    colour_selector = QuantizedColourSelector(colours)
+    colour_selector = QuantizedColourSelector(colours, max_value=0)
```

Comment:
I assume you plan to fix this properly (consider if you need a max_value parameter) but you just wanted to fix the error so it runs